lib/repl.js

```diff
@@ -1216,10 +1213,7 @@ REPLServer.prototype.resetContext = function() {
 REPLServer.prototype.displayPrompt = function(preserveCursor) {
   let prompt = this._initialPrompt;
   if (this[kBufferedCommandSymbol].length) {
-    prompt = '|';
-    const len = this.lines.level.length ? this.lines.level.length - 1 : 0;
-    const levelInd = StringPrototypeRepeat('..', len);
-    prompt += levelInd + ' ';
```

Comment:
Should we indent the line with spaces in this case? I guess it is actually handled differently now and the removal makes things easier and is correct therefore?